demos/custom-fields/pitch/custom-fields.css

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2019 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+ #notePicker {
+    background-image: url(media/notes.png);
+    border: 1px solid #ccc;
+    height: 109px;
+    width: 46px;
+}
```

Comment:
This file's name doesn't reflect that it is specific to the note picker.  Thus it should be called `pitch.css`.
Same issue present with turtle/custom-fields.css